conf.py

```diff
@@ -0,0 +1,133 @@
+"""
+    ReST directive for embedding Youtube and Vimeo videos.
+    There are two directives added: ``youtube`` and ``vimeo``. The only
+    argument is the video id of the video to include.
+    Both directives have three optional arguments: ``height``, ``width``
+    and ``align``. Default height is 281 and default width is 500.
+    Example::
+        .. youtube:: anwy2MPT5RE
+            :height: 315
+            :width: 560
+            :align: left
+    :copyright: (c) 2012 by Danilo Bargen.
+    :license: BSD 3-clause
+"""
+from __future__ import absolute_import
+from docutils import nodes
+from docutils.parsers.rst import Directive, directives
+
+# -- Project information -----------------------------------------------------
+
+project = 'Qubes OS'
+copyright = '2023, Qubes OS Project'
+author = 'Qubes OS Project'
+
+title = "Qubes Docs"
+html_title = "Qubes Docs"
+
+# The full version, including alpha/beta/rc tags
+release = '0.1'
+
+# -- General configuration ---------------------------------------------------
+
+html_static_path = ['attachment/doc']
+extensions = [
+    'sphinx.ext.autosectionlabel',
+    'sphinxnotes.strike',
+    'sphinx_reredirects'
+]
+
+redirects = {
+    "user/hardware/hcl": "https://www.qubes-os.org/hcl/",
+    "user/downloading-installing-upgrading/downloads:mirrors":"https://www.qubes-os.org/downloads/mirrors/",
+    "developer/general/visual-style-guide": "https://www.qubes-os.org/doc/visual-style-guide/",
+    "developer/general/website-style-guide":"https://www.qubes-os.org/doc/website-style-guide/",
+    "user/downloading-installing-upgrading/downloads":"https://www.qubes-os.org/downloads/",
+    "developer/general/how-to-edit-the-documentation":"https://www.qubes-os.org/doc/how-to-edit-the-documentation/"
+}
+
+autosectionlabel_prefix_document = True
+
+source_suffix = {
+    '.rst': 'restructuredtext',
+}
+templates_path = ['_templates']
+
+root_doc = "index"
+exclude_patterns = [
+    '_dev/*',
+    'attachment/*',
+    '**/*.txt'
+]
+
+html_theme = 'sphinx_rtd_theme'
+# html_theme = 'default'
+# html_theme = 'classic'
+
+html_theme_options = {
+    'externalrefs': True,
+    'bgcolor': 'white',
+    'linkcolor': '#99bfff',
+    'textcolor': '#000000',
+    'visitedlinkcolor': '#7b7b7b',
+    'bodyfont': '"Open Sans", Arial, sans-serif',
+    'codebgcolor': '$color-qube-light',
+    'codebgcolor': 'grey',
+    'body_min_width': '50%',
+    'body_max_width': '90%',
+    'collapse_navigation': True,
+}
+
+gettext_uuid = True
+gettext_compact = False
+
+# epub_show_urls = 'footnote'
+# latex_show_urls ='footnote'
+
+
+locale_dirs = ['_translated']
+
```

Comment:
> the option `language = 'en'` could be set explicitly

Won't that interfere with language set in readthedocs?

Reply:
> > the option `language = 'en'` could be set explicitly
> 
> Won't that interfere with language set in readthedocs?

I don't know. In the last minutes, I tried to find an answer on the Internet but without success. However, since [`language` defaults to `en`](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language), we can omit it.